docs/setup_wsl.md

```diff
@@ -250,6 +253,28 @@ Microsoft maintains an extensive [troubleshooting page](https://learn.microsoft.
 
 If you forgot you WSL password, follow the steps [here](https://learn.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-username-and-password) to reset it. Scroll down to the part beginnning with "If you forgot the password for yoru Linux distribution:".
 
+### Nuclear Option
+If you've tried everything and want to delete your WSL install entirely.
+
+<div class="primer-spec-callout danger" markdown="1">
+**WARNING:** This will delete all files inside WSL.
+</div>
+
+Start PowerShell and run it as administrator ([instructions](#install-wsl)).
+
+See what's installed.
+```console
+C:\WINDOWS\system32> wsl -l -v
+  NAME                   STATE           VERSION
+* Ubuntu-24.04           Running         2
+```
+
+Delete.
+```console
+$ wsl --unregister ubuntu
```

Comment:
I believe running both of these will result in an error message. Either the "pointer" from `Ubuntu` to `Ubuntu-24.04` does exist and it will delete 24.04, meaning the second one fails with an error message, or it doesn't exist and the first one fails with an error message. It doesn't cause any harm, but students may interpret it as something going wrong.

Would there be downsides to leaving just the second command `wsl --unregister Ubuntu-24.04` (and the note that their version may be different)?

(Also, if we do end up keeping the first command in some capacity, I think it should be capitalized `Ubuntu` rather than `ubuntu`, although I'm not sure if it is actually case-sensitive.)